awswrangler/s3/_read_text.py

```diff
@@ -360,7 +360,7 @@ def read_csv(
         last_modified_begin=last_modified_begin,
         last_modified_end=last_modified_end,
         ignore_index=ignore_index,
-        parallelism=parallelism,
+        ray_modin_args=ray_modin_args if ray_modin_args else {},
```

Comment:
Perhaps this check can just move to `_read_text_format` to avoid doing it for each `read_text` method?